ethereum/contracts/TestUtils.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity ^0.8.5 .0;
 
-import "hardhat/console.sol";
+//import "hardhat/console.sol";
```

Comment:
I don't see anything using this file. Can I just remove it?

Reply:
yeah sure